experimental/golden_response_checker_test.go

```diff
@@ -78,3 +78,12 @@ func TestGoldenResponseChecker(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestReadGoldenFile(t *testing.T) {
+	t.Run("read a long golden file", func(t *testing.T) {
+		goldenFile := filepath.Join("testdata", "long.golden.txt")
```

Comment:
```suggestion
		goldenFile := filepath.Join("testdata", "large.golden.txt")
```

maybe large better than long?  it is a long line :shrug:

Reply:
🤦